app/gateway/2.7.x/admin-api/index.md

```diff
@@ -1659,9 +1659,12 @@ following attributes must be set:
 * For `https`, at least one of `methods`, `hosts`, `headers`, `paths` or `snis`;
 * For `tcp`, at least one of `sources` or `destinations`;
 * For `tls`, at least one of `sources`, `destinations` or `snis`;
+* For `tls_passthrough`, must set `snis`;
```

Comment:
This file is generated from the code repo, please make the changes here instead: https://github.com/Kong/kong/blob/master/autodoc/admin-api/data/admin-api.lua#L857

Reply:
Opened https://github.com/Kong/kong/pull/8099